### PR TITLE
chore: use new c-sharp help category

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Quincy set up a [new subform](https://forum.freecodecamp.org/t/new-c-subforum/633423) for the C# cert. This PR adds the appropriate help category to point posts there.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
@moT01 it doesn't look like we show the "ask for help" button on any of the C# pages - do we maybe want it on the [trophy pages](https://www.freecodecamp.org/learn/foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/trophy-write-your-first-code-using-c-sharp)?

**Comment by moT01** [comment]
Yes, we could add it to all of the challenges @naomi-lgbt. It was requested that they be removed, but now requested to add it back since there isn't a default way to seek help on these.

**Comment by raisedadead** [comment]
Please expand tests (non-negotiable given the criticality of the cert), and this should be ready to go in.

**Comment by naomi-lgbt** [comment]
Tests added!

File: cypress/e2e/default/learn/challenges/c-sharp.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+describe('C Sharp Challenges', () => {
+  before(() => {
+    cy.visit(cSharpLocations.challenge);
+  });
+
+  it('renders the correct buttons', () => {
+    cy.get('.btn-primary').should('have.length', 2);
+    cy.contains('.btn-primary', 'Check your answer').should('exist');
+    cy.contains('.btn-primary', 'Ask for Help').should('exist');
```
Comment (ojeytonwilliams) [inline cypress/e2e/default/learn/challenges/c-sharp.ts:16]:
```suggestion
    cy.contains('.btn-primary', 'Check your answer').should('be.visible');
    cy.contains('.btn-primary', 'Ask for Help').should('be.visible');
```
`contains` already checks if the thing exists, so you can either leave it at that or check it's visible (which is a higher bar than existence)

File: cypress/e2e/default/learn/challenges/c-sharp.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  before(() => {
+    cy.login();
+    cy.visit(cSharpLocations.trophy);
+  });
+
+  it('renders the correct buttons', () => {
+    cy.get('.btn-primary').should('have.length', 3);
+    cy.contains('.btn-primary', 'Link Account').should('exist');
+    cy.contains('.btn-primary', 'Verify Trophy').should('exist');
+    cy.contains('.btn-primary', 'Ask for Help').should('exist');
```
Comment (ojeytonwilliams) [inline cypress/e2e/default/learn/challenges/c-sharp.ts:30]:
```suggestion
    cy.contains('.btn-primary', 'Link Account').should('be.visible');
    cy.contains('.btn-primary', 'Verify Trophy').should('be.visible');
    cy.contains('.btn-primary', 'Ask for Help').should('be.visible');
```

**Review by ojeytonwilliams — APPROVED** [review]
Nitpicks about the tests aside, this looks good!

**Review by raisedadead — APPROVED** [review]
Unblocking as well. Naomi, please land this and deploy.

**Pull request merged**